web/react/components/file_upload.jsx

```diff
@@ -280,7 +293,7 @@ export default class FileUpload extends React.Component {
                     ref='fileInput'
                     type='file'
                     onChange={this.handleChange}
-                    multiple='true'
+                    {...inputProps}
```

Comment:
Nope, I can get rid of it. I had that in there from some other stuff I was doing with the input while testing and just never took it out.